### PR TITLE
Release v1.13.0

### LINKED ISSUE #471: Refactor DicomFileRunner AddToReports
This issue relates to #430 and the new `_ignoreTextLessThan` setting.  As currently implemented this feature will ignore not just pixel text issues but also any tag where the failure is in a column with text data in it less than X length.  Also when run in standalone mode the pixel data reporter (`PixelTextFailureReport _tesseractReport`) will still report short values since it short circuits `AddToReports` and is used directly
```
protected override void AddToReports(Failure f)
        {
            if (_ignoreTextLessThan != 0 && f.ProblemValue.Length < _ignoreTextLessThan)
            {
                _logger.Debug($"Ignoring failure from resource {f.Resource} of length {f.ProblemValue.Length}");
                return;
            }

            base.AddToReports(f);
        }
```

### PR DESCRIPTION
**Pull request by rkm**
## Proposed Changes

See the CHANGELOG for release details.

## Checklist

By opening this PR, I confirm that I have:

- [x] Reviewed the [contributing](https://github.com/SMI/SmiServices/blob/master/CONTRIBUTING.md) guidelines for this repository
- [x] Ensured that the PR branch is in sync with the target branch (i.e. it is automatically merge-able)
- [x] Updated any relevant API documentation
- [x] Created or updated any tests if relevant
- [x] Accurately updated the [CHANGELOG](https://github.com/SMI/SmiServices/blob/master/CHANGELOG.md)
  - NOTE: This ***must*** include any changes to any of the following files: default.yaml, any of the RabbitMQ server configurations, GlobalOptions.cs
- [x] Listed myself in the [CONTRIBUTORS](https://github.com/SMI/SmiServices/blob/master/CONTRIBUTORS.md) file 🚀
- [x] Requested a review by one of the repository maintainers

## Issues
-   Closes #471

### CODE REVIEW

**Comment by rkm** [comment]
Willing to merge this if AppVeyor goes green...

**Comment by rkm** [comment]
@tznind apologies, I think you need to hit the approve button again before we can merge this :)

**Review by tznind — APPROVED** [review]

**Pull request merged**
